learningequality/kolibri-design-system
# Fix shifting `div` from `KTextbox`

## Description

Changes the `v-if` to the `v-show` in the KeenUI textbox so that the element is already in the DOM and we can toggle its `display`.

This is useful because it will keep that `div` below `KTextbox` for error validation, and prevent the UI from "shifting" whether or not there is an error validation for the field. 

#### Issue addressed
- Addresses #238 in KDS
- Addresses https://github.com/learningequality/kolibri/issues/8675
- Must work with: https://github.com/learningequality/kolibri/pull/8746 

### Before/after screenshots
See https://github.com/learningequality/kolibri/pull/8746 

## Steps to test

After linking KDS to Kolibri, see the following for testing: https://github.com/learningequality/kolibri/pull/8746

### Does this introduce any tech-debt items?

- We will want to double check that the places in Kolibri that this affects (see: https://github.com/learningequality/kolibri/pull/8746) are the only places where the shifting needs to be fixed.
- How can we document this, if this is how we will continue to deal with this issue in this way? Or, what can be done to be a more permanent fix?

## Testing checklist



## Reviewer guidance


- [ ] Is the code clean and well-commented (commented via `git commit`)

## Post-merge updates and tracking


## Comments
